egs/tedlium/s5_r3/run.sh

```diff
@@ -0,0 +1,224 @@
+  for dset in dev test; do
+    steps/decode.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+      exp/tri1/graph_nosp data/${dset} exp/tri1/decode_nosp_${dset}
+    steps/lmrescore_const_arpa.sh  --cmd "$decode_cmd" data/lang_nosp data/lang_nosp_rescore \
+       data/${dset} exp/tri1/decode_nosp_${dset} exp/tri1/decode_nosp_${dset}_rescore
+  done
+fi
+
+if [ $stage -le 11 ]; then
+  steps/align_si.sh --nj $nj --cmd "$train_cmd" \
+    data/train data/lang_nosp exp/tri1 exp/tri1_ali
+
+  steps/train_lda_mllt.sh --cmd "$train_cmd" \
+    4000 50000 data/train data/lang_nosp exp/tri1_ali exp/tri2
+fi
+
+if [ $stage -le 12 ]; then
+  utils/mkgraph.sh data/lang_nosp exp/tri2 exp/tri2/graph_nosp
+  for dset in dev test; do
+    steps/decode.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+      exp/tri2/graph_nosp data/${dset} exp/tri2/decode_nosp_${dset}
+    steps/lmrescore_const_arpa.sh  --cmd "$decode_cmd" data/lang_nosp data/lang_nosp_rescore \
+       data/${dset} exp/tri2/decode_nosp_${dset} exp/tri2/decode_nosp_${dset}_rescore
+  done
+fi
+
+if [ $stage -le 13 ]; then
+  steps/get_prons.sh --cmd "$train_cmd" data/train data/lang_nosp exp/tri2
+  utils/dict_dir_add_pronprobs.sh --max-normalize true \
+    data/local/dict_nosp exp/tri2/pron_counts_nowb.txt \
+    exp/tri2/sil_counts_nowb.txt \
+    exp/tri2/pron_bigram_counts_nowb.txt data/local/dict
+fi
+
+if [ $stage -le 14 ]; then
+  utils/prepare_lang.sh data/local/dict "<unk>" data/local/lang data/lang
+  cp -rT data/lang data/lang_rescore
+  cp data/lang_nosp/G.fst data/lang/
+  cp data/lang_nosp_rescore/G.carpa data/lang_rescore/
+
+  utils/mkgraph.sh data/lang exp/tri2 exp/tri2/graph
+
+  for dset in dev test; do
+    steps/decode.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+      exp/tri2/graph data/${dset} exp/tri2/decode_${dset}
+    steps/lmrescore_const_arpa.sh --cmd "$decode_cmd" data/lang data/lang_rescore \
+       data/${dset} exp/tri2/decode_${dset} exp/tri2/decode_${dset}_rescore
+  done
+fi
+
+if [ $stage -le 15 ]; then
+  steps/align_si.sh --nj $nj --cmd "$train_cmd" \
+    data/train data/lang exp/tri2 exp/tri2_ali
+
+  steps/train_sat.sh --cmd "$train_cmd" \
+    5000 100000 data/train data/lang exp/tri2_ali exp/tri3
+
+  utils/mkgraph.sh data/lang exp/tri3 exp/tri3/graph
+
+  for dset in dev test; do
+    steps/decode_fmllr.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+      exp/tri3/graph data/${dset} exp/tri3/decode_${dset}
+    steps/lmrescore_const_arpa.sh --cmd "$decode_cmd" data/lang data/lang_rescore \
+       data/${dset} exp/tri3/decode_${dset} exp/tri3/decode_${dset}_rescore
+  done
+fi
+
+if [ $stage -le 16 ]; then
+  # this does some data-cleaning.  It actually degrades the GMM-level results
+  # slightly, but the cleaned data should be useful when we add the neural net and chain
+  # systems.  If not we'll remove this stage.
+  local/run_cleanup_segmentation.sh
```

Comment:
did you ever test whether this data cleanup is helpful?

Reply:
Not yet.